submissions/Olumide Jokotade/HTML-CSS/Frere-wears-homepage.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html>
+    <head>
+      <link rel="stylesheet" type="text/css" href="Homepage.css"/>
+        <title>Frere Wears</title>
+    </head>
+    <body>
+        <h1>Welcome to Frere Wears</h1>
+        <h3><i>The home of Classic, Stylish and Bespoke Articles of Clothing for All men of All Ages, Classes and for All Seasons. Gentlemen, shall we?</i></h3>
+        <div class="topnav">
+                <a class="active" href="#home">Home</a>
+                <a href="#about">About us</a>
+                <a href="#contact">Contact us</a>
+                <a href="New Arrivals.html">New Arrivals</a> 
+                <a href="Registration.html">Create Account</a>
+                <a href="Login.html">Login</a>
+                <div class="search-container">
+                  <form action="/action_page.php">
+                    <input type="text" placeholder="Search..." name="search">
+                    <button type="submit"><i class="fa fa-search"></i></button>
+                  </form>
+                </div>
+         </div> 
+              <hr>
+         <ul>
+                    <li> <h4><a href="#Categories">Categories</a></h4></li>
+                    <li><a href="#Shirts">Shirts</a></li>
+                    <li><a href="#Turtlenecks">Turtlenecks</a></li>
+                    <li><a href="Pants">Pants and Shorts</a></li>
+                    <li><a href="Ties">Bespoke Ties</a></li>
+                    <li><a href="Tees">Tees</a></li>
+                    <li><a href="Footwears">Footwears</a></li>
+                    <li><a href="Ankara Designs">Ankara Designs</a></li>
+                    <li><a href="Underwears">Underwears</a></li>
+                    <li><a href="Jeans">Jeans</a></li>
+          </ul>
+                  <hr>
+          <div class="row">
+               <div class="column" class="container">
+                   <img 
+                      src="C:\Users\Jokotade\Desktop\Dufuna-CodeCamp19\submissions\Olumide Jokotade\HTML-CSS/Image10.jpg" 
```

Comment:
Use relative paths for these image files, as this will only work on your computer

Reply:
Could you please explain what you mean by relative path?